frontend/src/Routes.js

```diff
@@ -18,10 +15,7 @@ export function Routes() {
       <Route path="/about" exact component={AboutPage} />
```

Comment:
Maybe it would be nice if we use constants for route paths, so it will be easier to use and also it would be a good practice for students.